### PR TITLE
feat: update `Brazilian Portuguese` translations

### PR DESCRIPTION
**Pull request by luska1337**
- Updated translations for the Brazilian Portuguese language (pt-BR).
- Working with the most recent version `2.1.4`

### CODE REVIEW

File: pt-br.json [hunk 1]
```diff
@@ -1,1794 +1,1806 @@
...
+            "CONFIRM": "Confirmar",
+            "DELETE": "Excluir",
+            "ANDOTHERS": "{username} + {amount} outros",
+            "PRIVATE_ACCOUNT": "Esta conta é privada",
+            "PRIVATE_ACCOUNT_DESCRIPTION": "Siga esta conta para ver suas fotos e vídeos.",
+            "ERROR": "Erro",
+            "USERNAME_TAKEN": "Nome de usuário já está em uso",
+            "UNKNOWN": "Erro desconhecido",
+            "UNKNOWN_ACCOUNT": "Conta não existe",
+            "INVALID_USERNAME": "Invalid username",
```
Comment (Maximus7474) [inline pt-br.json:1113]: I've added some of the missing fields added in the 2.1.8 update, I'll let you translate these and I'll proceed with the merge.

**Review by Maximus7474 — CHANGES_REQUESTED** [review]

**Comment by Maximus7474** [comment]
Thanks for your contribution !

File: pt-br.json (hunk 1 above)
Comment (luska1337) [reply pt-br.json]: The translation was correct, I just made some increments and translated the `INVALID_USERNAME`

**Review by luska1337 — COMMENTED** [review]
update

**Comment by Maximus7474** [comment]
Thanks for your contribution !
I'm going ahead with the merge.

**Review by Maximus7474 — APPROVED** [review]

**Pull request merged**